source/aggregation-pipeline-builder.txt

```diff
@@ -85,6 +85,31 @@ To save your pipeline:
 
 #. Click :guilabel:`Save`.
 
+Compass stores saved pipelines in the following directory depending on
+your operating system:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 40
+   
+   * - Operating System
+     - File Location
+
+   * - macOS
+     - ``~/Library/Application Support/<Compass Edition>/FavoriteQueries``
+
+   * - Windows
+     - ``%APPDATA%/Roaming/<Compass Edition>/FavoriteQueries``
```

Comment:
Not sure if `Roaming` is correct?

Reply:
I don't think `Roaming` should be there. From my tests, `%APPDATA%` already points to the `Roaming` directory.

Reply:
Updated. I also just merged #235 to fix the other instance of `Roaming`